fud2/tests/tests.rs

```diff
@@ -70,23 +84,77 @@ fn req_slug(driver: &Driver, req: &Request) -> String {
     desc
 }
 
-fn test_emit(driver: &Driver, req: Request) {
+fn test_emit(driver: &Driver, req: Request, tag: &str) {
     let desc = req_desc(driver, &req);
     let slug = req_slug(driver, &req);
     let ninja = emit_ninja(driver, req);
     insta::with_settings!({
         description => desc,
```

Comment:
Perhaps the description for these "single-op" tests should also be different? Right now, it sorta assumes the request is based on states, so the description looks like "state1 -> state2". Maybe it would be more informative to just name the state that we are testing.